src/lightning_app/cli/lightning_cli.py

```diff
@@ -328,6 +330,12 @@ def run() -> None:
     default=False,
     help="run environment setup commands from the app comments.",
 )
+@click.option(
```

Comment:
Do we plan to support more auth ? If yes, would you mind listing them so we can derive a general API ?

Reply:
Yeah, in the plans we should add OAuth, where an app creator can allow access for all authenticated users or specify a restricted list of usernames (specified as sub-parameters) at least. Maybe, it can look like a `--enable-oauth --all-authenticated-users`/ `--enable-oauth --users-list john,doe,johndoe` e.g.